concent_api/utils/logging.py

```diff
@@ -220,3 +219,50 @@ def get_subtask_id_for_logging(message):
     if not isinstance(subtask_id, str):
         subtask_id = ''
     return subtask_id
+
+
+def log_gatekeeper_endpoint_request_received(request, operation: str):
```

Comment:
Why don't you use `replace_element_to_unavailable_instead_of_none` a decorator here?

Reply:
Because I log two variables:
-  operation, it can't be none. This method is called 2 times and this value is always given. 
- path_to_file. I get it from request and handle it in this method. 
Use of this decorator would work, but I don't think it's necessary